resources/views/api/emails/voucher_duplicate_email_textonly.blade.php

```diff
@@ -0,0 +1,22 @@
+* Voucher Duplicate Entered *
+
+==============================================
+
+[xXx]
+Hi {{ config('mail.to_admin.name') }},
+
+{{ $user }} has tried to submit voucher
+{{ $vouchercode }} against
+{{ $trader }} of
+{{ $market->name ?? 'no associated market' }}'s account, however that voucher has already been submitted by another trader.
```

Comment:
You have to assign it like `$this>market = $this->market ? $this->market->name : 'no associated market';`

Then use `$market` in the tpl like `$trader` and `$user`.